Update mac-schedule-scan.md (#7400)
MicrosoftDocs/microsoft-365-docs: microsoft-365/security/defender-endpoint/mac-schedule-scan.md

    launchctl start <your file name>
    ```

> 3. Your scheduled scan will run at the date, time, and frequency you defined in your p-list. In the examples above, the scan runs at 2:50 AM every Friday. 

Reviewer: ```suggestion
3. Your scheduled scan will run at the date, time, and frequency you defined in your p-list. In the previous examples, the scan runs at 2:50 AM every Friday. 
```

>     The `Weekday` value of `StartCalendarInterval` uses an integer to indicate the fifth day of the week, or Friday. The Range is between  0-7 and 7 represents sunday.

Reviewer: ```suggestion
    - The `Weekday` value of `StartCalendarInterval` uses an integer to indicate the fifth day of the week, or Friday. The range is between 0 and 7 with 7 representing Sunday.
```

>     The `Day` value of `StartCalendarInterval` uses an integer to indicate the third day of the month. The range is between 1-31.

Reviewer: ```suggestion
    - The `Day` value of `StartCalendarInterval` uses an integer to indicate the third day of the month. The range is between 1 and 31.
```

>     The `Hour` value of `StartCalendarInterval` uses an integer to indicate the second hour of the day. The range is between 0-24.

Reviewer: ```suggestion
    - The `Hour` value of `StartCalendarInterval` uses an integer to indicate the second hour of the day. The range is between 0 and 24.
```

>     The `Minute` value of `StartCalendarInterval` uses an integer to indicate Fivetly minutes of the Hour. The range is between 0-59.

Reviewer: ```suggestion
    The `Minute` value of `StartCalendarInterval` uses an integer to indicate fifty minutes of the hour. The range is between 0 and 59.
```
